Tools/debug/CollectSystemInfo.ps1

```diff
@@ -260,4 +282,7 @@ try {
 }
 
 Remove-Item -Path $progressFile -ErrorAction SilentlyContinue
-Write-InformationToArchive
+Write-InformationToArchive -FolderPath $folderPath -SubFolderPath $logfolderPath -ArchiveFileName $logfolderName
+if ($IncludeSensitiveData) {
+    Write-InformationToArchive -FolderPath $folderPath -SubFolderPath $dumpfolderPath -ArchiveFileName $dumpfolderName
```

Comment:
Will it work correctly if there is no dump file collected?

Reply:
Yes, I just want to be sure that there will be no errors in case if there are no dump files.

Reply:
uh, if so, the answer gonna be sure. there is no error if there are no dump files. folder and archive zip package will be empty then.